nixpkgs_review/cli/__init__.py

```diff
@@ -223,6 +224,11 @@ def common_flags() -> List[CommonFlag]:
             default=read_github_token(),
             help="Github access token (optional if request limit exceeds)",
         ),
+        CommonFlag(
+            "--nom",
+            action="store_true",
```

Comment:
Should this be just enabled by default if `nom` is in $PATH?

Reply:
I think this could become a string option pointing to the nom executable that defaults to the nom discovered in `$PATH`. If someone wants to disable nom instead they would than pass an empty string to this option.